src/caretogether-pwa/src/Families/FamilyScreen.tsx

```diff
@@ -386,6 +402,23 @@ export function FamilyScreen() {
               </Grid>}
           </Grid>
           <Grid container spacing={0}>
+			<Grid item xs={12}>
+              <h3 style={{ marginBottom: 0 }}>Communities</h3>
+				{familyCommunityInfo?.map(communityInfo => {
+					return (
+						<Card variant="outlined" sx={{ paddingBottom: '1rem' }}>
+							<CardHeader 
```

Comment:
Can you apply the Communities icon (the one that's used in the main nav menu) to the `avatar` prop, to give this card a helpful icon? Similar to [the "R" in the docs](https://mui.com/material-ui/react-card/#complex-interaction), but I think you can pass the icon without an `<Avatar>` component.

Reply:
Icon added